### ISSUE TITLE
454107EF - FIFA 07

### ISSUE THREAD

**Issue by TotalCaesar659** [xenia-project/game-compatibility#633]
[Marketplace](http://marketplace.xbox.com/en-US/Product/FIFA-07/66acd000-77fe-1000-9115-d802454107ef)

Tested on (https://github.com/benvanik/xenia/commit/52c75c8dbc253d3ddd5e602b307ff4e0ca5d95bf)

# Issues:
Instant crash.

# Log:
[Log file](https://drive.google.com/file/d/0B7KpdfS_wYnZQllpdk1DUmd4aVU/view)

# Screenshot(s):
![0000](https://cloud.githubusercontent.com/assets/14265316/18257935/d01589c4-73d4-11e6-9d88-d39c52ae02d4.jpg)

# Labels:
(state-crash)

**Comment by Wolffel** [comment]
I have the same problem with this game: 
[xenia.log](https://github.com/xenia-project/game-compatibility/files/4603986/xenia.log)

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/733efa5ff8a8f1c45da32df9e3e6f6a26bc5dd6f

Watched a CPU vs CPU match for a 25 ingame minutes. 120+FPS during matches, 30ish FPS on replays with vsync off and shaders cached. One of the few FIFA games that looks and runs really well! Probabaly Playable.

:grey_question:Audio sometimes squeaks?

![fifo (2)](https://user-images.githubusercontent.com/31154606/103105909-96404d80-45f6-11eb-8c71-6c16d5cd2262.png)

<details>
  <summary>More Screenshots</summary>

![fifo (1)](https://user-images.githubusercontent.com/31154606/103105908-96404d80-45f6-11eb-96de-ab6bf1d49c21.png)
![fifo (3)](https://user-images.githubusercontent.com/31154606/103105910-96d8e400-45f6-11eb-82c5-e63411ebe66f.png)
![fifo (4)](https://user-images.githubusercontent.com/31154606/103105911-96d8e400-45f6-11eb-8675-0504dead45ca.png)
![fifo (5)](https://user-images.githubusercontent.com/31154606/103105912-97717a80-45f6-11eb-8571-332ce02489bf.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5740483/xenia.zip)

state-gameplay

**Comment by v3rc3ttj** [comment]
still crashes when playing at night, just can play at day. When it's crashes at night, it must reset window to play it.

**Comment by Walker-24** [comment]
I tried playing it and it worked fine the first time. After closing and launching again the emulator I can't play any match, the game crashes while loading

**Comment by kudozer** [comment]
> Tested on [xenia-project/xenia@733efa5](https://github.com/xenia-project/xenia/commit/733efa5ff8a8f1c45da32df9e3e6f6a26bc5dd6f)
> 
> Watched a CPU vs CPU match for a 25 ingame minutes. 120+FPS during matches, 30ish FPS on replays with vsync off and shaders cached. One of the few FIFA games that looks and runs really well! Probabaly Playable.
> 
> ❔Audio sometimes squeaks?
> 
> ![fifo (2)](https://user-images.githubusercontent.com/31154606/103105909-96404d80-45f6-11eb-8c71-6c16d5cd2262.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5740483/xenia.zip)
> 
> state-gameplay

How did you tweak the config file to play the game? Thanks !

**Comment by kudozer** [comment]
> Tested on [xenia-project/xenia@733efa5](https://github.com/xenia-project/xenia/commit/733efa5ff8a8f1c45da32df9e3e6f6a26bc5dd6f)
> 
> Watched a CPU vs CPU match for a 25 ingame minutes. 120+FPS during matches, 30ish FPS on replays with vsync off and shaders cached. One of the few FIFA games that looks and runs really well! Probabaly Playable.
> 
> ❔Audio sometimes squeaks?
> 
> ![fifo (2)](https://user-images.githubusercontent.com/31154606/103105909-96404d80-45f6-11eb-8c71-6c16d5cd2262.png)
> 
> More Screenshots
> [xenia.zip](https://github.com/xenia-project/game-compatibility/files/5740483/xenia.zip)
> 
> state-gameplay

How did you tweak the config file to play the game? Mine is ok at first - menu, selection & everything but always crashes right before starting a match...

**Comment by AhayriSG** [comment]
@v3rc3ttj "just can play at day" => not anymore with recent versions.
![fifa07crash](https://user-images.githubusercontent.com/59465544/179206692-38bdb5d4-1297-4b40-8c3e-0c1e2d9bc096.gif)

RTV/DSV=same btw.

**Comment by Maarcis12** [comment]
yeah, i can concur with @AhayriSG FIFA 07 still crashes upon loading a match even on recent canary build.

**Comment by Sowa95** [comment]
Tested on: https://github.com/xenia-project/xenia/commit/f6b5424a9f7daee69ea7ea837b6d4224ea47c368

[crash_before_match.zip](https://github.com/xenia-project/game-compatibility/files/14890630/crash_before_match.zip)
[gameplay.zip](https://github.com/xenia-project/game-compatibility/files/14890632/gameplay.zip)

- still crashes before entering a match (preventable).
- bad audio with crackles, looping, cutting out (use canary).
- lights are not occluded.

Played one 6 min half, night works as well.

Crash when loading to match probably timing related. Before starting a match, set single core affinity for xenia.exe in task manager. Once _LOADING.....PLEASE WAIT!_ pops up, restore default core affinity.
Perhaps `time_scalar` can be used instead, but no success on my machine.

<details><summary>Night match screenshots</summary>
<p>

![2](https://github.com/xenia-project/game-compatibility/assets/113144725/ef7fcc27-c9ae-44ac-9e10-67b6c6826033)
![3](https://github.com/xenia-project/game-compatibility/assets/113144725/d2454262-d90f-4dc4-b1f7-4ea980596105)

</p>
</details>

**Comment by Eddef1** [comment]
I got it to work without any crash, from previous comment

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/107
